envs/dandi/terraform.tfvars

```diff
@@ -3,3 +3,4 @@ singleuser_image_repo = "dandiarchive/dandihub"
 region = "us-east-2"
 singleuser_image_tag = "latest"
 jupyterhub_domain = "hub.dandiarchive.org"
+eks_cluster_version = "1.32"
```

Comment:
Do we need to update this [value](https://github.com/dandi/dandi-hub/blob/b6f1007d6063d06d14678a0c804d5a1be21dd766/variables.tf#L19) too?

Reply:
No that's just the default